### PR TITLE
Add initial documentation for new SMLIGHT SLZB integration

### PR DESCRIPTION
**Pull request by tl-sl**
## Proposed change

This PR provides initial documentation for new SMLIGHT integration for SLZB-06 Zigbee adapters.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/118675
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/5530
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
  - Introduced SMLIGHT SLZB integration for Home Assistant.
  - Enabled monitoring and managing of SLZB-06x devices.
  - Added support for automation and access to SMLIGHT web UI features.
  - New sensors for core temperature, Zigbee temperature, core uptime, Zigbee uptime, RAM usage, and filesystem usage.

- **Documentation**
  - Comprehensive documentation added for the SMLIGHT SLZB integration, detailing setup requirements and device compatibility.

### CODE REVIEW

File: source/_integrations/smlight.markdown [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ha_domain: smlight
+ha_zeroconf: true
+ha_platforms:
+  - sensor
+ha_codeowners:
+  - '@tl-sl'
+ha_integration_type: device
+---
+
+The [SMLIGHT](https://smlight.tech) SLZB-06x ethernet Zigbee coordinators
```
Comment (c0ffeeca7) [inline source/_integrations/smlight.markdown]:
```suggestion
The [SMLIGHT](https://smlight.tech) SLZB-06x Ethernet Zigbee coordinators
```

File: source/_integrations/smlight.markdown [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+provide a convenient way to add Zigbee to your smart home setup.
+
+This integration allows users to monitor and manage their SLZB-06x devices
```
Comment (c0ffeeca7) [inline source/_integrations/smlight.markdown]:
```suggestion
The **SMLIGHT SLZB** {% term integrations %} allows users to monitor and manage their SLZB-06x devices
```

File: source/_integrations/smlight.markdown [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+directly from within Home Assistant. Allowing for direct access to many of the
```
Comment (c0ffeeca7) [inline source/_integrations/smlight.markdown]:
```suggestion
directly from within Home Assistant and to directly access many of the
```

File: source/_integrations/smlight.markdown [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+features found in the SMLIGHT web UI. You can also use these in your automations.
+
+{% include integrations/config_flow.md %}
+
+## Prerequisite
+
+A supported SLZB-06 Adapter, this integration has been tested with:
```
Comment (c0ffeeca7) [inline source/_integrations/smlight.markdown]:
```suggestion
You need a supported SLZB-06 adapter.

 This integration has been tested with the following devices:
```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]

**Comment by tl-sl** [comment]
Thanks for the review and suggestions, I have applied these to this PR now.

File: source/_integrations/smlight.markdown [hunk 5]
```diff
@@ -0,0 +1,47 @@
+---
+title: SMLIGHT SLZB
+description: The SMLIGHT SLZB integration allows users to monitor and manage their SMLIGHT SLZB-06x devices from directly within Home Assistant.
+ha_category:
+  - Sensor
+ha_release: 2024.07.0
```
Comment (c0ffeeca7) [inline source/_integrations/smlight.markdown]:
```suggestion
ha_release: 2024.9
```

File: source/_integrations/smlight.markdown [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+  - sensor
+ha_codeowners:
+  - '@tl-sl'
+ha_integration_type: device
+---
+
+The [SMLIGHT](https://smlight.tech) SLZB-06x Ethernet Zigbee coordinators
+provide a convenient way to add Zigbee to your smart home setup.
+
+The **SMLIGHT SLZB** {% term integrations %} allows users to monitor and manage their SLZB-06x devices
```
Comment (c0ffeeca7) [inline source/_integrations/smlight.markdown]:
```suggestion
The **SMLIGHT SLZB** {% term integration %} allows users to monitor and manage their SLZB-06x devices
```

File: source/_integrations/smlight.markdown [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+directly from within Home Assistant and to directly access many of the
+features found in the SMLIGHT web UI. You can also use these in your automations.
+
+{% include integrations/config_flow.md %}
```
Comment (c0ffeeca7) [inline source/_integrations/smlight.markdown]:
```suggestion
```

File: source/_integrations/smlight.markdown [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+You need a supported SLZB-06 adapter.
+
+ This integration has been tested with the following devices:
+
+- [SLZB-06](https://smlight.tech/product/slzb-06)
+- [SLZB-06M](https://smlight.tech/product/slzb-06m)
+- [SLZB-06p7](https://smlight.tech/product/slzb-06p7)
+- [SLZB-06p10](https://smlight.tech/product/slzb-06p10/)
+
```
Comment (c0ffeeca7) [inline source/_integrations/smlight.markdown:35]:
```suggestion
  
{% include integrations/config_flow.md %}
```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]

File: source/_integrations/smlight.markdown [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+You need a supported SLZB-06 adapter.
+
+ This integration has been tested with the following devices:
+
+- [SLZB-06](https://smlight.tech/product/slzb-06)
+- [SLZB-06M](https://smlight.tech/product/slzb-06m)
+- [SLZB-06p7](https://smlight.tech/product/slzb-06p7)
+- [SLZB-06p10](https://smlight.tech/product/slzb-06p10/)
+  
+{% include integrations/config_flow.md %}
```
Comment (c0ffeeca7) [inline source/_integrations/smlight.markdown:36]:
```suggestion
{% include integrations/config_flow.md %}

```

File: source/_integrations/smlight.markdown [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+---
+
+The [SMLIGHT](https://smlight.tech) SLZB-06x Ethernet Zigbee coordinators
+provide a convenient way to add Zigbee to your smart home setup.
+
+The **SMLIGHT SLZB** {% term integration %} allows users to monitor and manage their SLZB-06x devices
+directly from within Home Assistant and to directly access many of the
+features found in the SMLIGHT web UI. You can also use these in your automations.
+
+## Prerequisite
```
Comment (c0ffeeca7) [inline source/_integrations/smlight.markdown]:
```suggestion
## Prerequisites
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @tl-sl :+1:

**Pull request merged**